app/lib/ui/components/theme_selector.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+import 'package:ods_flutter_demo/model_theme.dart';
+
+const List<String> list = <String>['System', 'ThemeMode.system', 'Three', 'Four'];
+
+class ThemeSelector extends StatefulWidget {
+  const ThemeSelector({super.key});
+
+  @override
+  State<ThemeSelector> createState() => _ThemeSelectorState();
+}
+
+class _ThemeSelectorState extends State<ThemeSelector> {
+  String themeValue = ThemeMode.system.toString();
+
+  @override
+  Widget build(BuildContext context) {
+    List<String> themeList = <String>[ThemeMode.light.toString(), ThemeMode.dark.toString(), ThemeMode.system.toString()];
+    return DropdownButton<String>(
+      value: themeValue,
+      icon: const Icon(Icons.arrow_downward),
+      elevation: 16,
+      onChanged: (String? value) {
+        setState(() {
+          themeValue = value!;
+        });
+        themeNotifier.themeMode = value == ThemeMode.system.toString() ? ThemeMode.system : (value == ThemeMode.dark.toString() ? ThemeMode.dark : ThemeMode.light);
```

Comment:
Works well except for this part. Need to find a way to access this `themeNotifier` (pass it to this component)